src/getCurrentPosition/getCurrentPosition.web.tsx

```diff
@@ -0,0 +1,15 @@
+type GetCurrentPositionOptions = {
+    maximumAge: number;
+    timeout: number;
+    enableHighAccuracy: boolean;
+};
+
+const getCurrentPosition = (success: (response: GeolocationPosition) => void, error: (error: GeolocationPositionError) => void, options: GetCurrentPositionOptions) => {
```

Comment:
where did `GeolocationPositionError` come from? 🙂 

Also, `getCurrentPosition` should have exactly the same interface on web and mobile, meaning that each parameter of the function should have exactly the same type.

The approach I took with `x-maps` (and the approach that is suggested in our TS guidelines) is to create a separate type file where shared types should be defined.

In this case, we can create `getCurrentPosition.types.ts` under  the `getCurrentPosition` directory. We can define the type for the `getCurrentPosition` function like this:

```ts
export type GetCurrentPosition= (success: SuccessCB error: ErrorCB options: GetCurrentPositionOptions) => void
```

assuming that the `SuccessCB`, `ErrorCB` and `GetCurrentPositionOptions` types are defined.

And then, in each platform file (`getCurrentPosition.ts` and `getCurrentPosition.web.ts`), you can import the newly defined type.

```ts
import { GetCurrentPosition } from './getCurrentPosition.types`;

const getCurrentPosition: GetCurrentPosition = (successDb, errorDb, options) => {...}
```